From foo@baz Fri Jun 28 11:10:15 PDT 2013
Date: Fri, 28 Jun 2013 11:10:15 -0700
To: Greg KH <gregkh@linuxfoundation.org>
Cc: Felipe Balbi <balbi@ti.com>
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: USB: phy: remove CONFIG_USB_DEBUG usage

Now that no USB phy driver is using CONFIG_USB_DEBUG, remove it from the
Makefile.

Cc: Felipe Balbi <balbi@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/usb/phy/Makefile |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/usb/phy/Makefile
+++ b/drivers/usb/phy/Makefile
@@ -1,9 +1,6 @@
 #
 # Makefile for physical layer USB drivers
 #
-
-ccflags-$(CONFIG_USB_DEBUG) := -DDEBUG
-
 obj-$(CONFIG_USB_PHY)			+= phy.o
 obj-$(CONFIG_OF)			+= of.o
 
